src/ra_rules.ruleset

```diff
@@ -1,11 +1,21 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <RuleSet Name="ra_rules" ToolsVersion="15.0">
-  <Include Path="cppcorecheckclassrules.ruleset" Action="Default" />
-  <Include Path="cppcorecheckconstrules.ruleset" Action="Default" />
-  <Include Path="cppcorecheckfuncrules.ruleset" Action="Default" />
-  <Include Path="cppcorechecksharedpointerrules.ruleset" Action="Default" />
-  <Include Path="cppcorecheckuniquepointerrules.ruleset" Action="Default" />
+  <Include Path="cppcorecheckclassrules.ruleset" Action="Warning" />
+  <Include Path="cppcorecheckconstrules.ruleset" Action="Warning" />
+  <Include Path="cppcorecheckfuncrules.ruleset" Action="Warning" />
+  <Include Path="cppcorechecksharedpointerrules.ruleset" Action="Warning" />
+  <Include Path="cppcorecheckuniquepointerrules.ruleset" Action="Warning" />
+  <Include Path="nativerecommendedrules.ruleset" Action="Warning" />
```

Comment:
I was originally trying to see if I could get these to show up without having a huge list at the bottom but it doesn't work. It shows up as checked in the UI but I guess the compiler doesn't understand it.